src/platforms/java/guides/spring-boot/logging-frameworks.mdx

```diff
@@ -67,7 +67,16 @@ However, if errors that may appear during startup should to be sent to Sentry, t
 
 ### Log4j2
 
-To use Sentry's Log4j2 integration in Spring Boot application, [follow the guide on configuring Log4j2 with Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-configure-log4j-for-logging) and configure `SentryAppender` in the `log4j2.xml` file:
+To use Sentry's Log4j2 integration in Spring Boot application, you must include a dependency to the `sentry-log4j` module:
```

Comment:
The module name should be: `sentry-log4j2`

Reply:
```suggestion
To use Sentry's Log4j2 integration in Spring Boot application, you must include a dependency to the `sentry-log4j2` module:
```